ansible-playbook/roles/gophish/tasks/main.yml

```diff
@@ -34,11 +39,55 @@
   apt:
     upgrade: safe
 
+- name: Ensure /etc/ssl/csr folder exists
+  file:
+    path: /etc/ssl/csr
+    state: directory
+    mode: '0755'
+  
+- name: Ensure /etc/ssl/private folder exists
+  file:
+    path: /etc/ssl/private
+    state: directory
+    mode: '0755'
+
+- name: Ensure /etc/ssl/crt folder exists
+  file:
+    path: /etc/ssl/crt
+    state: directory
+    mode: '0755'
+
+- name: adding existing user '{{ gophish_user }}' to group ssl-cert
```

Comment:
Hey @PaulSec! For whatever reason, I'm getting an error that the `ssl-cert` group doesn't exist. However, if I move the package installation step before this one, things work great. Can we make that change, then we should be in good shape to merge!

Reply:
Haha, all good then! (it's weird it worked out of the box for me on a Debian 10). All resolved :)